satnogsclient/scheduler/tasks.py

```diff
@@ -30,9 +30,19 @@ def spawn_observation(*args, **kwargs):
 
 def get_jobs():
     """Query SatNOGS Network API to GET jobs."""
+
+    print 'Try getting Jobs from Network ..'
+
     url = urljoin(settings.NETWORK_API_URL, 'jobs')
     params = {'ground_station': settings.GROUND_STATION_ID}
-    response = requests.get(url, params=params)
+    try:
+        response = requests.get(url, params=params, verify=settings.CA_CERT)
+    except:
+        print 'HTTPs certificate verification failed..(ignore for now)'
+        response = requests.get(url, params=params, verify=False)
```

Comment:
Instead of messing with SSL cert files, we can add a quick workaround with ``verify=False`` to avoid any SSL checks. This is going to be temporary since we are fixing this in the network level.